packages/components/autocomplete/src/autocomplete.vue

```diff
@@ -89,10 +90,16 @@
 </template>
 
 <script lang="ts" setup>
-import { ref, computed, onMounted, nextTick } from 'vue'
+import {
+  ref,
+  computed,
+  onMounted,
+  nextTick,
+  useAttrs as useCompAttrs,
+} from 'vue'
 import { debounce } from 'lodash-unified'
+import { OnClickOutside } from '@vueuse/components'
```

Comment:
Why not use 
```import { onClickOutside } from '@vueuse/core'```

Reply:
I feel a little troublesome，use renderless component will be easier
![image](https://user-images.githubusercontent.com/39730999/155297787-ddeb6289-c14a-416b-a92f-bccca00b5211.png)

Reply:
But this will add a extra dependency

Reply:
Yeah~you're right, I'll remove it now